modules/serverless-eventing/pages/service-mesh/eventing-service-mesh-sinkbinding.adoc

```diff
@@ -0,0 +1,165 @@
+= Using SinkBinding with {SMProductName}
+:compat-mode!:
+// Metadata:
+:description: Using SinkBinding with {SMProductName}
+
+.Prerequisites
+
+* You have followed the setup {SMProductShortName} with {ServerlessProductName} procedure
+
+.Procedure
+
+. Create a `Service` in a namespace that is member of the `ServiceMeshMemberRoll`:
++
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: event-display
+  namespace: <namespace> <1>
+spec:
+  template:
+    metadata:
+      annotations:
+        sidecar.istio.io/inject: "true" <2>
+        sidecar.istio.io/rewriteAppHTTPProbers: "true"
+    spec:
+      containers:
+      - image: quay.io/openshift-knative/knative-eventing-sources-event-display:latest
```

Comment:
I just change the image and that's it? I don't think this is a critical change, so I'd do that afterwards if we need to, since that means I need to retest the examples

Reply:
It probably needs bigger changes. Let's do this later. I've put a comment on https://github.com/openshift/openshift-docs/pull/60772#issuecomment-1588766605 to not forget about it.